tests/test_jira_download.py

```diff
@@ -90,18 +100,24 @@ def one_bad_issue_callback(self, request, context):
     @classmethod
     def setUpClass(cls):
         cls.jira_connection = get_connection()
-        with open("test_data/jira/test_issues_response.json", "r") as issues_file:
+        with open("tests/test_data/jira/test_issues_response.json", "r") as issues_file:
```

Comment:
@thingscouldbeworse - most of the files changes here are formatting, but this was an intentional change. For some reason this base file path was missing in this test. It was causing errors for me when run on the command line